telemetry_package/main.py

```diff
@@ -1,31 +1,140 @@
-import sys
+'''
+This code gets too many messages all at once. Hence it needs to: 
+    - The Node only sends the first message it receives
+'''
+
+
+import json
+import os
+import select
+import socket
+# import sys
 import rclpy
 from rclpy.node import Node
+from rcl_interfaces.msg import Log
+
+
+DEFAULT_UNIX_SOCKET_PATH = "/tmp/thi_drone"
+
+
+class MinimalPublisher(Node):
```

Comment:
Please change it back to derive from CommonNode.
```suggestion
from common_package_py.common_node import CommonNode
class TelemetryNode(CommonNode):
```

Reply:
Do we want to have an heartbeat from the telemetry node.
I thought we decided to make it a non requirement for continuing a mission.

Reply:
Well you're right about that. It comes from a time where it was also responsible for saving the logs in a file.